tests/lib/tools/setup_nested_hybrid_system.sh

```diff
@@ -304,7 +324,7 @@ main() {
 
     run_muinstaller "${model_assertion}" "${store_dir}" "${gadget_snap}" \
         "${gadget_assertion}" "${kernel_snap}" "${kernel_assertion}" "${label}" \
-        "${disk}"
+        "${disk}" "${passphrase}" "${extra_muinstaller_args}"
```

Comment:
And then related to a previous command about `shift`, use a list for `extra_muinstaller_args`.